### PR TITLE
Remove deprecated fields.

### PR DESCRIPTION
**Pull request by JRBANCEL**
These prevents `latest` from being installed.

### CODE REVIEW

**Comment by ZhiminXiang** [comment]
Will the auto TLS not enabled? Or it will still be enabled without these explicit configuration?

**Comment by JRBANCEL** [comment]
This 
```
  meshConfig:
    defaultConfig:
      proxyMetadata:
        TERMINATION_DRAIN_DURATION_SECONDS: "20"
```
seems enough.

**Comment by JRBANCEL** [comment]
A `PeerAuthentication` seems required now. Let me test this.

**Review by nak3 — APPROVED** [review]

File: third_party/istio-latest/install-istio.sh [hunk 1]
```diff
@@ -29,6 +29,20 @@ tar xzf ${ISTIO_TARBALL}
 # Install Istio
 ./istio-${ISTIO_VERSION}/bin/istioctl install -f "$(dirname $0)/$1"
 
+# Enable mTLS STRICT
+if [[ -n "$KIND" ]]; then
+  cat <<EOF | kubectl apply -f -
```
Comment (nak3) [inline third_party/istio-latest/install-istio.sh:34]: I personally prefer to putting a yaml file and run `kubectl apply -f "$(dirname $0)/extra/global-mtls.yaml"`. But either is fine.

Comment (tcnghia) [reply third_party/istio-latest/install-istio.sh:34]: +1 to moving yaml outside of `bash`. already easy to make mistakes with bash without embedding :D

**Review by tcnghia — APPROVED** [review]

**Review by tcnghia — APPROVED** [review]
/lgtm

**Pull request merged**